frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/request-vehicles/simulated-region-overview-behavior-request-vehicles.component.ts

```diff
@@ -0,0 +1,161 @@
+import type { OnChanges } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type {
+    DelayEventActivityState,
+    RequestBehaviorState,
+} from 'digital-fuesim-manv-shared';
+import {
+    isWaitingForAnswer,
+    UUID,
+    SimulatedRegionRequestTargetConfiguration,
+    TraineesRequestTargetConfiguration,
+} from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import { map, combineLatest } from 'rxjs';
+import { ExerciseService } from 'src/app/core/exercise.service';
+import type { AppState } from 'src/app/state/app.state';
+import {
+    createSelectActivityStates,
+    createSelectBehaviorState,
+    selectSimulatedRegions,
+    selectCurrentTime,
+} from 'src/app/state/application/selectors/exercise.selectors';
+
+type RequestTargetOption = UUID | 'trainees';
+
+@Component({
+    selector: 'app-simulated-region-overview-behavior-request-vehicles',
+    templateUrl:
+        './simulated-region-overview-behavior-request-vehicles.component.html',
+    styleUrls: [
+        './simulated-region-overview-behavior-request-vehicles.component.scss',
+    ],
+})
+export class RequestVehiclesComponent implements OnChanges {
+    @Input() simulatedRegionId!: UUID;
+    @Input() requestBehaviorId!: UUID;
+
+    requestBehaviorState$!: Observable<RequestBehaviorState>;
+
+    requestTargetOptions$!: Observable<{
+        [key in RequestTargetOption]: string;
+    }>;
+
+    waitingForAnswer$!: Observable<boolean>;
+
+    nextTimeoutIn$!: Observable<number>;
+
+    initialRequestTargetOption$!: Observable<RequestTargetOption>;
```

Comment:
Why does the name of this observable include "initial"?

Reply:
I've changed it in 103c3d3ddcd06afd35c9ce1aa574a9bf3fa3771e